examples/pygeosxExamples/hydraulicFractureWithMonitor/hydraulicFractureWithMonitor.py

```diff
@@ -1,41 +1,41 @@
 
-# PYGEOSX_SETUP
+# PYGEOS_SETUP
 from mpi4py import MPI
-import pygeosx
-from pygeosx_tools import wrapper
+import pygeos
```

Comment:
I thoght there was an issue with an existing python package that has this same name. @cssherman ?

Reply:
Yes, and it is a pre-requisite to a commonly used package that manages map projections (see https://pygeos.readthedocs.io/en/stable/ and https://pyproj4.github.io/pyproj/stable/).

Reply:
Also `pySomething` or `something-py` are usually used for wrappers around code of other languages. If I understood correctly `pygeos` is it exactly a wrapper around geos. It is more an orchestrator. What we have is more a `rungeos` in python

Reply:
ah...ok. well...I wasn't sure about changing the name of the python stuff. What are your suggestions @cssherman ?

Reply:
Maybe `geos-wrapper` or `geos-python-driver`?

Reply:
`geos-py-wrapper`  in case someday someone wants a `geos-lua-wrapper`... By the way, in this last case, it would require to add lua folder next to each python folder.

May I suggest to pull out all these python mixed C stuff into a new language mixed package ? 
It should have `geos` as a external dependency.

Reply:
I went with `geos_python_interface` from within python, and `geosPythonInterface` within `geos`. Now that I write that out, perhaps `geos_interface` from within python and `pythonInterface` within `geos` would be better....and a little less redundant.

Reply:
I had linking problems when changing the python names...I reverted and will do that after we resolve the circular dependency.